tests/unit/test_utils.py

```diff
@@ -227,16 +227,17 @@ def test_explicit_pager(self):
 
     def test_exit_of_context_manager_for_pager(self):
         self.set_session_pager('mypager --option')
-        with self.stream_factory.get_pager_stream():
+        with self.stream_factory.get_pager_stream() as stream:
+            stream.write()
```

Comment:
Why do we need an empty write now?

Reply:
To create a real stream  since it's lazy now.